bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/scheduler/ThreadPoolManager.java

```diff
@@ -5,16 +5,25 @@
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
  */
-package org.eclipse.smarthome.core.common;
+package org.eclipse.smarthome.core.scheduler;
```

Comment:
I think option b is probably the quickest intermediate solution then. So could you move the class back and squash all your commits? I can then try to take over further processing (probably soon creating a CQ as this might also take a while to be approved). Thanks!